modules/gitops-openshift-go-common-terms.adoc

```diff
@@ -0,0 +1,144 @@
+// Module included in the following assemblies:
+//
+// * understanding_openshift_gitops/about-redhat-openshift-gitops.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="gitops-openshift-go-common-terms_{context}"]
+= Glossary of common terms for OpenShift {gitops-shortname}
+
+This glossary defines common OpenShift {gitops-shortname} terms.
+
+Application Controller (Argo CD Application Controller)::
+A controller that continuously watches the Git repository for changes, monitors running applications, compares the live state against the desired target state, and deploys new changes. Examples include Argo CD Application Controller  detecting OutOfSync application state and optionally taking corrective action.
```

Comment:
```suggestion
A controller that continuously watches the Git repository for changes, monitors running applications, compares the live state against the desired target state, and deploys new changes. Examples include Argo CD the Application Controller detecting an OutOfSync application state and optionally taking corrective action.
```

Reply:
changed: Examples include Argo CD the Application Controller detecting an OutOfSync application state and optionally taking corrective action.